src/ui/deleteAccount/ViewDeleteAccount.qml

```diff
@@ -80,25 +103,22 @@ Item {
 
         State {
             name: "StateVerificationSessionByTotpNeeded"
-            when: VPNAuthInApp.state === VPNAuthInApp.StateVerificationSessionByTotpNeeded || VPNAuthInApp.state === VPNAuthInApp.StateVerifyingSessionTotpCode
+            when: (
+                VPNAuthInApp.state === VPNAuthInApp.StateVerificationSessionByTotpNeeded
+                || VPNAuthInApp.state === VPNAuthInApp.StateVerifyingSessionTotpCode
+            )
             PropertyChanges {
                 target: loader
                 source: "../authenticationInApp/ViewAuthenticationVerificationSessionByTotpNeeded.qml"
             }
         },
 
-        State {
-            name: "StateFallbackInBrowser"
-            when: VPNAuthInApp.state === VPNAuthInApp.StateFallbackInBrowser
```

Comment:
Maybe add a comment about these remaining states and why we do not cover them here.

Reply:
Added a comment above the `states` property.